app/lib/shared/markdown.dart

```diff
@@ -63,6 +69,17 @@ class _HashLink implements m.NodeVisitor {
   }
 }
 
+/// Enables inline <br> to be emitted as <br/> HTML element.
+class _InlineBrHtmlSyntax extends m.TextSyntax {
+  _InlineBrHtmlSyntax() : super(r'(<br>)|(<br\s*/>)|(<BR>)|(<BR\s*/>)');
```

Comment:
Wouldn't it be better to:
A) move this upstream, and ensure that the `markdown` package has a safe subset of inline-HTML supported?
B) handle more generic cases? ie. a whitelist of permitted tags or something like that?

Reply:
It could be better code-wise, but the effort/benefit doesn't yield us that much.

Reply:
So far there is one package that has this problem, it's not like property HTML rendering is an urgent feature we need to deliver :)

Reading on package `markdown` it refers to: https://github.com/showdownjs/showdown/wiki/Markdown%27s-XSS-Vulnerability-(and-how-to-mitigate-it)

Which leads me to think we need a separate package for sanitizing HTML... hmm..

Reply:
The `[some text](javascript:alert('xss'))` gets us, but the rest is a non-issue, as we don't allow inline HTMLs (besides this `<br>`)